### PR TITLE
Cht academy overview doc

### PR DESCRIPTION
**Pull request by esthermmoturi**
Draft PR for CHT academy documentation

- [x] Add documentation on course navigation

### CODE REVIEW

File: content/en/apps/tutorials/local-setup.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ To install a valid certificate, open a terminal in the `cht-core` directory. Ens
 ```shell
 ./scripts/add-local-ip-certs-to-docker-4.x.sh cht_nginx_1
 ```
+If you are using an older version of cht-core, ensure that the  `add-local-ip-certs-to-docker.sh` file is available under the scripts folder.
```
Comment (craig-landry) [inline content/en/apps/tutorials/local-setup.md:192]: This line looks unrelated to the concept of an overview. Please remove it from this PR and open a separate one so we can review separately and keep things clear in the git history.

File: content/en/apps/tutorials/cht-academy.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Overview of CHT Academy"
+linkTitle: CHT Academy
+weight: 5
```
Comment (craig-landry) [inline content/en/apps/tutorials/cht-academy.md:4]: Please make sure this shows up first within the Tutorials section. I think it makes sense for people to see this before other tutorial content.

**Review by craig-landry — CHANGES_REQUESTED** [review]
Sometimes I see pages and sections like this and they feel unnecessary, but then I'll end up seeing people struggle to find where to start. It's easy for us to assume everyone will get it, but that's often not true. With that, thanks for adding this stuff! I think it'll make more of a difference than we might expect.

I made a few small comments, but know this is a draft, so I assume things will get tidied up. In general: good stuff!

**Comment by nsitaula** [comment]
Hi @esthermmoturi is there anything needed from me? I noticed @craig-landry reviewed and suggested some updates.

**Pull request closed without merging**